### DISCUSSION TITLE
New 0.2.x release?

### DISCUSSION THREAD

**Discussion by tronical** [cross-rs/cross#1165] [General]
Hi!

First of all: Cross is wonderful! :)

I was wondering if there are any plans for a new release soon, in particular of the docker images? Or is the plan to go straight to 0.3 for the next release?

The "latest" tag is quite a few months old and I see that there have been much-needed upgrades to the containers. I can work around this by using the "edge"/"main" tags, but it would be nice to go back to a released version :)

**Comment by Alexhuszagh** [comment]
It would be the 0.3 release: numerous breaking changes, especially in the container images themselves, have been introduced, such as updating the base Ubuntu versions, glibc versions, and much more.

You can see the backlog on the [0.3 milestone](https://github.com/cross-rs/cross/milestone/2).

**Reply by tronical** [reply]
Thanks for the quick response. I'll follow the milestone.

**Comment by guihkx** [comment]
Now that 0.2.5 is released, I was really hoping that #1142 would be included. :(

It would fix [this build error](https://github.com/guihkx/topgrade/actions/runs/4092921685/jobs/7057990732) I'm experiencing when targeting DragonFly BSD...

**Reply by Emilgardis** [reply]
Oh my bad! I could've included that :/

**Reply by guihkx** [reply]
It's okay, I can wait for the next release :)
